docs/vwii/vwii-modding.md

```diff
@@ -2,97 +2,78 @@
-1. Launch the Homebrew Launcher, by opening the Mii Maker.
-1. Launch the compat_installer.
-1. Press `A` to install the Homebrew Channel and wait until you see `Install succeeded`. Then press the HOME button to return to the Wii U Menu.
-1. Launch vWii (the Wii Menu icon).
-   - If the installation has succeeded, you should see the Homebrew Channel in your Wii Menu.
+1. Boot into Aroma
+1. Launch compat_installer from the Home Menu
+1. Press `A` to install the Homebrew Channel and wait until you see `Install succeeded`, then press the HOME button to return to the Wii U Menu
+1. Launch your vWii (the Wii Menu icon)
+   - If the installation has succeeded, you should see the Homebrew Channel installed on your Wii Menu
 
 ### Installing cIOSes
 
 !> Make sure you have no `.wad` files anywhere else than in the `apps` folder on your SD Card.
 
-1. Power on your Wii U and launch vWii.
-1. Launch The Homebrew Channel.
-1. Launch d2x cIOS Installer.
+1. Power on your Wii U and launch the vWii
+1. Launch The Homebrew Channel
+1. Launch d2x cIOS Installer
 1. Set everything to match the following:
     - Select cIOS: `d2x-v11-beta1-vWii`
     - Select cIOS base: `56`
     - Select cIOS slot: `249`
-1. Press the `A` button to install.
+1. Press the `A` button to install
 1. Set everything to match the following:
     - Select cIOS: `d2x-v11-beta1-vWii`
     - Select cIOS base: `57`
     - Select cIOS slot: `250`
-1. Press the `A` button to install.
+1. Press the `A` button to install
 1. Set everything to match the following:
     - Select cIOS: `d2x-v11-beta1-vWii`
     - Select cIOS base: `58`
     - Select cIOS slot: `251`
-1. Press the `A` button to install.
-1. Press the `B` button to exit.
+1. Press the `A` button to install
+1. Press the `B` button to exit
 
 ### Patching IOS 80
 
-!> If for any reason, the Wii U is turned off while patching IOS 80, your vWii will be bricked. This can be fixed by either extracting the IOS 80 from your previously made NAND backup and then replacing it over FTP or by [reinstalling IOS 80](recover-vwii-ioses-channels).
+!> If for any reason the Wii U is turned off while patching IOS 80, your vWii will be bricked. This can be fixed by either extracting the IOS 80 from your previously made NAND backup and then replacing it over FTP, or by [reinstalling IOS 80](recover-vwii-ioses-channels).
 
-1. Launch Patched IOS 80 Installer for vWii.
-1. Read the warning screen and wait 30 seconds.
-1. Press any buttons to install.
-1. Wait until it says <code>IOS80 <wbr>Installation <wbr>is <wbr>complete!</code>.
-1. Press any buttons to exit.
+1. Launch Patched IOS 80 Installer for vWii
+1. Read the warning screen and wait 30 seconds
+1. Press any buttons to install
+1. Wait until it says <code>IOS80 <wbr>Installation <wbr>is <wbr>complete!</code>
+1. Press any buttons to exit
+
+!> Priiloader versions before 0.10.0 do not work with vWii. Installing versions of Priiloader before 0.8.2 will brick your vWii.

```

Comment:
These two sentences seem to imply the exact same thing. Could it be a bit more concise? 

Such as
`Do not install any Priiloader version prior to 0.10.0, or it will brick your vWii.`
?

Reply:
I worded it that way specifically since 0.8.2-0.9.1 would refuse to install vwii, while anything before 0.8.2 had zero checks and would install priiloader, then bricking vwii.

Reply:
I see. We can leave it alone in that case.